remote/web/package.json

```diff
@@ -6,25 +6,25 @@
 		"@microsoft/1ds-core-js": "^3.2.13",
 		"@microsoft/1ds-post-js": "^3.2.13",
 		"@vscode/iconv-lite-umd": "0.7.0",
+		"@vscode/tree-sitter-wasm": "^0.0.2",
 		"@vscode/vscode-languagedetection": "1.0.21",
-		"@xterm/addon-clipboard": "0.2.0-beta.34",
-		"@xterm/addon-image": "0.9.0-beta.51",
-		"@xterm/addon-search": "0.16.0-beta.51",
-		"@xterm/addon-serialize": "0.14.0-beta.51",
-		"@xterm/addon-unicode11": "0.9.0-beta.51",
-		"@xterm/addon-webgl": "0.19.0-beta.51",
+		"@xterm/addon-clipboard": "0.2.0-beta.35",
+		"@xterm/addon-image": "0.9.0-beta.52",
+		"@xterm/addon-search": "0.16.0-beta.52",
+		"@xterm/addon-serialize": "0.14.0-beta.52",
+		"@xterm/addon-unicode11": "0.9.0-beta.52",
+		"@xterm/addon-webgl": "0.19.0-beta.52",
+		"@xterm/xterm": "5.6.0-beta.52",
 		"@xterm/xterm": "5.6.0-beta.51",
-		"he": "^1.2.0",
```

Comment:
We lost `he`! Can we please bring `he` back? We'll need to `yarn` so the reh-web `package.json` can re-inherit `he` as well

Reply:
done!